### ISSUE TITLE
Clarify special columns in API docs

### ISSUE THREAD

**Issue by jkbradley** [graphframes/graphframes#50]
We should make it clear what happens when a vertex or edge DataFrame contains a column with a special name.  E.g., what happens when you call PageRank on a graph whose vertices already have a "pagerank" column?  We currently throw an error, but we should be more explicit about this, both in runtime checks in the code and in the API docs.

**Comment by felixcheung** [comment]
And possibly '__tmp'

**Comment by jkbradley** [comment]
Good point.  __tmp should probably be chosen dynamically (appending something to avoid conflicts)
